www/js/services.js

```diff
@@ -8,17 +8,15 @@ angular.module('starter.services', [])
         viewAs : 'hymn',
         font : 'gurbaniakhar',
         search : 'gurmukhi_search'
-    }
+    };
 
     if(data.length===0)
     {
-      Store.set(store,defaults);
-      return Store.get(store);
-    }
-    else
-    {
-      return Store.get(store);
+      data = defaults;
```

Comment:
Surely this file should be restored to it's original otherwise the defaults will always over-ride any customizations the user makes?